[2024-07-23T19:08:27.952+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-23T19:08:28.006+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hackaton_dag.hackaton_task manual__2024-07-23T17:36:50.396578+00:00 [queued]>
[2024-07-23T19:08:28.025+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hackaton_dag.hackaton_task manual__2024-07-23T17:36:50.396578+00:00 [queued]>
[2024-07-23T19:08:28.029+0000] {taskinstance.py:2306} INFO - Starting attempt 4 of 4
[2024-07-23T19:08:28.049+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): hackaton_task> on 2024-07-23 17:36:50.396578+00:00
[2024-07-23T19:08:28.063+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=301) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-23T19:08:28.062+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'hackaton_dag', 'hackaton_task', 'manual__2024-07-23T17:36:50.396578+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/dag_script.py', '--cfg-path', '/tmp/tmpxt_hvuhu']
[2024-07-23T19:08:28.067+0000] {standard_task_runner.py:64} INFO - Started process 303 to run task
[2024-07-23T19:08:28.067+0000] {standard_task_runner.py:91} INFO - Job 16: Subtask hackaton_task
[2024-07-23T19:08:28.130+0000] {task_command.py:426} INFO - Running <TaskInstance: hackaton_dag.hackaton_task manual__2024-07-23T17:36:50.396578+00:00 [running]> on host beb4e5ba0df4
[2024-07-23T19:08:28.474+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='hackaton_dag' AIRFLOW_CTX_TASK_ID='hackaton_task' AIRFLOW_CTX_EXECUTION_DATE='2024-07-23T17:36:50.396578+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-23T17:36:50.396578+00:00'
[2024-07-23T19:08:28.477+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-23T19:08:28.504+0000] {etl_script.py:7} INFO - Starting data extraction process
[2024-07-23T19:08:28.505+0000] {helper_functions.py:14} INFO - Fetching data from https://restcountries.com/v3.1/all
[2024-07-23T19:08:30.900+0000] {helper_functions.py:16} INFO - Data fetched successfully from https://restcountries.com/v3.1/all
[2024-07-23T19:08:30.923+0000] {helper_functions.py:76} INFO - Transforming data
[2024-07-23T19:08:30.937+0000] {helper_functions.py:87} INFO - Data transformation complete
[2024-07-23T19:08:30.939+0000] {helper_functions.py:93} INFO - Transforming languages
[2024-07-23T19:08:30.962+0000] {helper_functions.py:100} INFO - Transformation complete
[2024-07-23T19:08:30.963+0000] {helper_functions.py:107} INFO - Cleaning data
[2024-07-23T19:08:30.968+0000] {helper_functions.py:121} INFO - Data cleaning complete
[2024-07-23T19:08:30.969+0000] {helper_functions.py:127} INFO - Loading data to PostgreSQL database
[2024-07-23T19:08:30.969+0000] {helper_functions.py:135} ERROR - Error loading data to PostgreSQL: invalid literal for int() with base 10: 'None'
[2024-07-23T19:08:30.970+0000] {etl_script.py:23} INFO - Data loading process complete
[2024-07-23T19:08:30.973+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-23T19:08:30.974+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-23T19:08:30.985+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=hackaton_dag, task_id=hackaton_task, run_id=manual__2024-07-23T17:36:50.396578+00:00, execution_date=20240723T173650, start_date=20240723T190828, end_date=20240723T190830
[2024-07-23T19:08:31.024+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-07-23T19:08:31.047+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-23T19:08:31.052+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
